sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCWriteSuite.scala

```diff
@@ -452,15 +452,18 @@ class JDBCWriteSuite extends SharedSparkSession with BeforeAndAfter {
     val df = spark.createDataFrame(sparkContext.parallelize(data), schema)
 
     // out-of-order
-    val expected1 = Map("id" -> "BIGINT", "first#name" -> "VARCHAR(123)", "city" -> "CHAR(20)")
+    val expected1 =
+      Map("id" -> "BIGINT", "first#name" -> "CHARACTER VARYING(123)", "city" -> "CHARACTER(20)")
```

Comment:
interesting. `varchar` is not a valid type name in h2 any more?

Reply:
Yes. H2 changed the type name.